docs/commands/index.md

```diff
@@ -8,6 +8,7 @@ Documentation commands:
 
 - [`preview-docs`](preview-docs.md) Preview API reference docs for the specified API description.
 - [`build-docs`](build-docs.md) Build API description into an HTML file.
+- [`preview`](preview.md) Start a local preview of a Redocly documentation project one of the product NPM packages.
```

Comment:
```suggestion
- [`preview`](preview.md) Start a local preview of a Redocly project one of the product NPM packages.
```

Reply:
(It's not limited to documentation. It also starts a local mock server, for example.)